tide_landing_page.install

```diff
@@ -738,3 +738,15 @@ function tide_landing_page_update_8016() {
     }
   }
 }
+
+/**
+ * Updates Call to action component to enable WYSIWYG.
+ */
+function tide_landing_page_update_8017() {
+  $field = FieldConfig::loadByName('paragraph', 'call_to_action', 'field_paragraph_body');
+  $settings = $field->getThirdPartySettings('allowed_formats');
+  if ($settings) {
+    $field->setThirdPartySetting('allowed_formats', 'rich_text', '0');
```

Comment:
How come the export above is referencing `plain_text`?

Reply:
@anthony-malkoun . sorry. changed.